diff --git debian/slapd.postinst debian/slapd.postinst
--- debian/slapd.postinst
+++ debian/slapd.postinst
@@ -48,7 +48,7 @@ postinst_upgrade_configuration() {					# {{{
 	fi
 
 	# Move to slapd.d configuration style.
-	migrate_to_slapd_d_style
+	# migrate_to_slapd_d_style
 
 	# One-time upgrade fix for olcAccess on cn=Subschema
 	if previous_version_older 2.4.23-5 && previous_version_newer 2.4.23-3 \
diff --git debian/slapd.scripts-common debian/slapd.scripts-common
--- debian/slapd.scripts-common
+++ debian/slapd.scripts-common
@@ -788,8 +788,7 @@ fi
 
 # Load the default location of the slapd config file
 if [ -z "$SLAPD_CONF" ]; then
-	if [ -f "/etc/ldap/slapd.conf" ] && \
-		[ ! -e "/etc/ldap/slapd.d" ]
+	if [ -f "/etc/ldap/slapd.conf" ]
 	then
 		SLAPD_CONF="/etc/ldap/slapd.conf"
 	else
diff --git debian/slapd.preinst debian/slapd.preinst
--- debian/slapd.preinst
+++ debian/slapd.preinst
@@ -116,6 +116,7 @@ if [ "$MODE" = upgrade ] || [ "$MODE" = install -a -n "$OLD_VERSION" ]; then
 fi
 
 if [ "$MODE" = upgrade ]; then
+	SLAPD_CONF=/etc/ldap/slapd.conf
 	dump_databases
 fi
 
