diff -Nur openldap-2.4.41+dfsg.o/debian/configure.options openldap-2.4.41+dfsg/debian/configure.options
--- openldap-2.4.41+dfsg.o/debian/configure.options	2015-08-11 11:37:14.000000000 +0200
+++ openldap-2.4.41+dfsg/debian/configure.options	2015-08-11 11:37:54.000000000 +0200
@@ -155,6 +155,7 @@
 #    --enable-translucent  Translucent Proxy overlay no|yes|mod [no]
 #    --enable-unique       Attribute Uniqueness overlay no|yes|mod [no]
 #    --enable-valsort      Value Sorting overlay no|yes|mod [no]
+#    --enable-translog     Transactions Log overlay no|yes|mod [no]
 #
 #Library Generation & Linking Options
 #  --enable-static[=PKGS]  build static libraries [default=yes]
diff -Nur openldap-2.4.41+dfsg.o/debian/slapd.install openldap-2.4.41+dfsg/debian/configure.options
--- openldap-2.4.41+dfsg.o/debian/slapd.install	2015-08-11 11:37:14.000000000 +0200
+++ openldap-2.4.41+dfsg/debian/slapd.install	2015-08-11 11:37:54.000000000 +0200
@@ -10000,0 +10000,3 @@
+usr/lib/ldap/translog-2.4.so*
+usr/lib/ldap/translog.so*
+usr/lib/ldap/translog.la
