  * add lost patch for core.schema (Bug #22468)

diff -Nuar -x '*.orig' openldap-2.4.31.orig/debian/schema/core.schema openldap-2.4.31/debian/schema/core.schema
--- openldap-2.4.31.orig/debian/schema/core.schema	2012-07-19 14:40:03.443920153 +0200
+++ openldap-2.4.31/debian/schema/core.schema	2012-07-19 14:41:49.456917589 +0200
@@ -199,7 +199,9 @@ attributetype ( 2.5.4.22 NAME 'teletexTerminalIdentifier'
 
 attributetype ( 2.5.4.23 NAME ( 'facsimileTelephoneNumber' 'fax' )
 	DESC 'RFC2256: Facsimile (Fax) Telephone Number'
-	SYNTAX 1.3.6.1.4.1.1466.115.121.1.22 )
+	EQUALITY caseIgnoreMatch
+	SUBSTR caseIgnoreSubstringsMatch
+	SYNTAX 1.3.6.1.4.1.1466.115.121.1.15{64} )
 
 attributetype ( 2.5.4.24 NAME 'x121Address'
 	DESC 'RFC2256: X.121 Address'
@@ -338,8 +340,8 @@ attributetype ( 2.5.4.48 NAME 'protocolInformation'
 
 attributetype ( 2.5.4.50 NAME 'uniqueMember'
 	DESC 'RFC2256: unique member of a group'
-	EQUALITY uniqueMemberMatch
-	SYNTAX 1.3.6.1.4.1.1466.115.121.1.34 )
+	EQUALITY distinguishedNameMatch
+	SUP distinguishedName )
 
 attributetype ( 2.5.4.51 NAME 'houseIdentifier'
 	DESC 'RFC2256: house identifier'
