Bug #36215 - deny posix logins if user password expiration date has been reached

diff --git debian/rules debian/rules
--- debian/rules
+++ debian/rules
@@ -23,7 +23,7 @@ ifneq ($(filter stage1,$(DEB_BUILD_PROFILES)),)
 	CONFIG += --disable-slapd
 endif
 
-CONTRIB_MODULES = autogroup lastbind passwd passwd/pbkdf2 passwd/sha2 smbk5pwd k5pwd pwd_scheme_kinit
+CONTRIB_MODULES = autogroup lastbind passwd passwd/pbkdf2 passwd/sha2 smbk5pwd k5pwd pwd_scheme_kinit shadowbind
 
 # Ensure CC is set correctly for cross builds, unless it has already 
 # been set explicitly.
diff --git debian/slapd.install debian/slapd.install
--- debian/slapd.install
+++ debian/slapd.install
@@ -10000,0 +10000,2 @@
+usr/lib/ldap/shadowbind.so*
+usr/lib/ldap/shadowbind.la
