diff --git /usr/share/univention-directory-notifier/univention-translog /usr/share/univention-directory-notifier/univention-translog
index ef65bb26be..88566e7c2b 100755
--- /usr/share/univention-directory-notifier/univention-translog
+++ /usr/share/univention-directory-notifier/univention-translog
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.7
+#!/usr/bin/env python3
 # vim: set fileencoding=utf-8 filetype=python foldmethod=expr expandtab shiftwidth=4 tabstop=4 :
 """
 Univention Directory Notifier Transaction log admin commmand
@@ -80,7 +80,7 @@ class IndexHeader(Structure):

         The header is architecture dependant due to the use of `c_ulong`.
     """
-    MAGIC = 0x3395e0d4L
+    MAGIC = 0x3395e0d4
     _fields_ = [("magic", c_ulong)]
     _pack_ = 1

@@ -1387,7 +1387,7 @@ def parse_size(string):
     suffix = string.lstrip(".0123456789")
     try:
         unit, = suffix.rstrip("iIbB").upper() or ' '
-        scale = 1L << (10 * " KMGTPE".index(unit))
+        scale = 1 << (10 * " KMGTPE".index(unit))
     except ValueError:
         raise ArgumentTypeError("Invalid unit")
